docs/sources/mimir/release-notes/v2.11.md

```diff
@@ -0,0 +1,92 @@
+- **Include a `Retry-After` header in recoverable error responses from the distributor.** This can protect your Mimir cluster from clients that default to retrying very quickly. Enable this feature by setting the `-distributor.retry-after-header.enabled` CLI flag.
+- **Improved query-scheduler performance under load** by using a central dispatcher to allocate requests to queriers instead of having each querier spend cycles checking for work to do on each incoming request.
+
+## Experimental features
+
+Grafana Mimir 2.11 includes new features that are considered experimental and disabled by default. Please use them with caution and report any issue you encounter:
+
+- **Block specified queries on a per-tenant basis.** This is configured via the `blocked_queries` limit.
+- **Store metadata when ingesting metrics via OTLP.** This makes metric description and type available when ingesting metrics via OTLP. You can enable this feature by setting the CLI flag `-distributor.enable-otlp-metadata-storage` to `true`.
+- **Reject gRPC push requests that the ingester/distributor is unable to accept before reading them into memory.** You can enable this feature by using the `-ingester.limit-inflight-requests-using-grpc-method-limiter` and/or the `-distributor.limit-inflight-requests-using-grpc-method-limiter` CLI flags for the ingester and/or the distributor, respectively.
+- **Customize the memcached client write and read buffer size.** The buffer allocated for each memcached connection can be configured via the following CLI flags:
+  - For the blocks storage:
+    - `-blocks-storage.bucket-store.chunks-cache.memcached.read-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.chunks-cache.memcached.write-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.index-cache.memcached.read-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.index-cache.memcached.write-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.metadata-cache.memcached.read-buffer-size-bytes`
+    - `-blocks-storage.bucket-store.metadata-cache.memcached.write-buffer-size-bytes`
+  - For the query frontend:
+    - `-query-frontend.results-cache.memcached.read-buffer-size-bytes`
+    - `-query-frontend.results-cache.memcached.write-buffer-size-bytes`
+  - For the ruler storage:
+    - `-ruler-storage.cache.memcached.read-buffer-size-bytes`
+    - `-ruler-storage.cache.memcached.write-buffer-size-bytes`
+- **Configure the number of long-living workers used to process gRPC requests.** This can decrease CPU usage by reducing the number of stack allocations. Configure this feature by using the `-server.grpc.num-workers` CLI flag.
+- **Enforce a limit in bytes on the `PostingsForMatchers()` cache used by ingesters.** This limit can be configured via the `-blocks-storage.tsdb.head-postings-for-matchers-cache-max-bytes` and `-blocks-storage.tsdb.block-postings-for-matchers-cache-max-bytes` CLI flags.
```

Comment:
[nit] Probably don't need the brackets here - should we even just call it "the postings for matchers cache"? 

```suggestion
- **Enforce a limit in bytes on the `PostingsForMatchers` cache used by ingesters.** This limit can be configured via the `-blocks-storage.tsdb.head-postings-for-matchers-cache-max-bytes` and `-blocks-storage.tsdb.block-postings-for-matchers-cache-max-bytes` CLI flags.
```